packages/node_modules/nav-frontend-skjema/src/textarea.js

```diff
@@ -23,6 +23,7 @@ class Textarea extends Component {
 
     componentDidMount() {
         requestAnimationFrame(this.updateHeight, 0);
+        window.addEventListener('resize', this.updateHeight, false);
     }
```

Comment:
Feilen skjer kun når man aktivt resizer vinduet, siden høyden på textarea ikke oppdateres med størrelsen på vinduet, men bredden endrer seg med resizing og textarea-elementet blir dermed for lite til å vise hele teksten på en gang (går utifra at dette er ønskelig siden vertikal scroll i textarea'n ikke funker). Løsningen ble derfor å kjøre this.updateHeight i textarea på window.resize.

Reply:
Da trenger du kanskje å fjerne eventListeneren også? Dersom dette er aktuelt for flere komponenter, kan det være greit å lage en wrapper-komponent som håndterer resize listeningen - da kan en evt. legge inn throttle og tilsvarende dersom det skulle trenges.